MetaMask/metamask-extension
# Send page components address book addresslistitem

## **Description**

Implements a new `AddressListItem` component which looks similar to the `AccountListItem` but is much lighter and does not attempt to show totals or anything other than a `label` and `address`

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1566

## **Manual testing steps**

1. `MULTICHAIN=1 yarn start`
2. Enter send flow
3. Look at the Contact List -- see this new component for each item
4. Click an item, see it populated to "To" field
5. Empty "To" field
6. In the "To" field, type "metamask.eth"
7. See search items look the same.

## **Screenshots/Recordings**

### **Before**

<img width="410" alt="SCR-20231027-lzap" src="https://github.com/MetaMask/metamask-extension/assets/46655/6515123c-c8a8-4ecb-94cc-89d98ec639b2">


### **After**

<img width="435" alt="SCR-20231027-lxjo" src="https://github.com/MetaMask/metamask-extension/assets/46655/31029b92-9c86-4266-b7c0-0da410ac2dbf">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
